### ISSUE TITLE
Not Automatically Reading config.yml

### ISSUE THREAD

**Issue by Zuberlan** [qaisjp/go-discord-irc#130]
The discord bridge running from a docker is not reading the config.yml file and updating the channels automatically.  The bot on discord has all the necessary privileges and can update Webhooks.

**Comment by qaisjp** [comment]
makes sense, this is because we copy the config.yaml and not pipe it as a volume - https://github.com/qaisjp/go-discord-irc/blob/a82661f3bda41da038c375da65e969d350091bd4/Dockerfile#L5
